doc/rest.md

```diff
@@ -0,0 +1,173 @@
+# REST
+
+When running and connected to a DotBot gateway, the `dotbot-controller`
+application provides a REST server to get information and control connected
```

Comment:
application provides a REST server to send commands to and receive information from connected DotBots.